### DISCUSSION TITLE
Manage issues and pull requests

### DISCUSSION THREAD

**Discussion by Akbaroke** [community/community#145622] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

What is the best way to manage issues and pull requests in an active open-source repository to keep collaboration between contributors efficient and organized?

**Comment by IKHINtech** [comment, marked as answer]
Managing issues and pull requests in an active open-source repository can be challenging, but a few best practices can help maintain efficiency and organization. Here are some tips:

1. Labeling Issues and Pull Requests: Use labels to indicate the category of the issue (such as bug, enhancement, or help wanted) and the status of the pull request (such as WIP, ready for review). These labels help contributors and maintainers understand the priority and status of each item.

2. Templates for Issues and Pull Requests: Providing templates for issues and pull requests makes it easy for new contributors to provide complete and clear information. These templates can include a description of the problem, steps to reproduce the bug, or instructions for code changes.

3. Set Review and Quick Response Rules: Set rules about response times for pull requests and issues. For example, specify that every pull request must be reviewed within 48 hours. This helps ensure that contributors don’t feel neglected and that development on the project continues.

4. Milestones and Roadmaps: Use the milestone feature to track progress and set priorities for new issues and features. This also provides a clear picture of what needs to be accomplished to achieve the project’s short- or long-term goals.

5. Close Completed Issues: Once a pull request has been approved and merged, be sure to close the associated issue. This helps keep the repository clean and organized and provides clear feedback on the progress that has been made.

**Reply by Akbaroke** [reply]
Thank you Ihin for the answer you gave, it really helped🙏
